src/components/layout/HeaderMenu.vue

```diff
@@ -12,21 +12,16 @@ export default {
   data() {
     return {
       menuItems: [
-        { label: this.$t("menu.contacts"), route: "/profiles/contacts" },
         {
-          label: this.$t("menu.trailsTreasuries"),
-          route: "/trails/treasuries",
+          label: this.$t("menu.daos"),
+          route: "/trails/ballots",
+          filter: "polls",
+        },
+        {
+          label: this.$t("menu.proposals"),
+          route: "/trails/ballots",
+          filter: "worker-proposals",
```

Comment:
We actually want more than just the `worker-proposals`. We want:
- `worker-proposals`
- `polls`

Reply:
@poplexity we discussed that `t-f-election` should be displayed on another page. I'm under the impression `amend-ballots` should be moved as well. Is that correct?

Reply:
@karynemayer we did discuss to remove TF elections from the menu for now, but prepare it to be a second menu entry next to "proposals" once activated at a point of time. This might also include the arbitration elections at some point.

Reply:
The header and left menus are fixed. And that is all about this issue.
but...

What should I do with this filter menu?
Should we always and only show proposals?
or polls as well?
What about referendums?
If there's something to change maybe is good to create another issue.

![image](https://user-images.githubusercontent.com/4420760/191126390-81618783-91d7-47cd-87ed-dd453d088427.png)

Reply:
@Kersten-TCD The issue for creating the Elections page is #94 
@Viterbo so, we want to have consistent behavior before anything else. So, it seems that if we clean `Status` and `Group`, there are no filters being applied. Thus, my suggestion is that we show all of the three types of proposals by default.